### ISSUE TITLE
Problem with Mongo Driver 3.5.0

### ISSUE THREAD

**Issue by ismaelgomescosta** [javers/javers#648]
The current version of Mongo Driver has a bug.

https://jira.mongodb.org/browse/JAVA-2590

Javers can`t instantiate when mongo is configured to use SSL connections.

**Comment by bartoszwalacik** [comment]
we use the same version as Spring Boot, maybe it's time to bump to latest 1.x Spring Boot

**Comment by bartoszwalacik** [comment]
Im afraid that the last 1.x spring-boot is still using old version of mongo java driver

**Comment by bartoszwalacik** [comment]
I can bump to 3.6.3 but spring-boot 1.x is still using 3.4.3. It could mean conflicts on the app side ...

```
+--- org.springframework.boot:spring-boot-starter-data-mongodb:1.5.10.RELEASE
|    +--- org.springframework.boot:spring-boot-starter:1.5.10.RELEASE
|    |    +--- org.springframework.boot:spring-boot:1.5.10.RELEASE
|    |    |    +--- org.springframework:spring-core:4.3.14.RELEASE
|    |    |    |    \--- commons-logging:commons-logging:1.2
|    |    |    \--- org.springframework:spring-context:4.3.14.RELEASE
|    |    |         +--- org.springframework:spring-aop:4.3.14.RELEASE
|    |    |         |    +--- org.springframework:spring-beans:4.3.14.RELEASE
|    |    |         |    |    \--- org.springframework:spring-core:4.3.14.RELEASE (*)
|    |    |         |    \--- org.springframework:spring-core:4.3.14.RELEASE (*)
|    |    |         +--- org.springframework:spring-beans:4.3.14.RELEASE (*)
|    |    |         +--- org.springframework:spring-core:4.3.14.RELEASE (*)
|    |    |         \--- org.springframework:spring-expression:4.3.14.RELEASE
|    |    |              \--- org.springframework:spring-core:4.3.14.RELEASE (*)
|    |    +--- org.springframework.boot:spring-boot-autoconfigure:1.5.10.RELEASE
|    |    |    \--- org.springframework.boot:spring-boot:1.5.10.RELEASE (*)
|    |    +--- org.springframework.boot:spring-boot-starter-logging:1.5.10.RELEASE
|    |    |    +--- ch.qos.logback:logback-classic:1.1.11
|    |    |    |    +--- ch.qos.logback:logback-core:1.1.11
|    |    |    |    \--- org.slf4j:slf4j-api:1.7.22 -> 1.7.25
|    |    |    +--- org.slf4j:jcl-over-slf4j:1.7.25
|    |    |    |    \--- org.slf4j:slf4j-api:1.7.25
|    |    |    +--- org.slf4j:jul-to-slf4j:1.7.25
|    |    |    |    \--- org.slf4j:slf4j-api:1.7.25
|    |    |    \--- org.slf4j:log4j-over-slf4j:1.7.25
|    |    |         \--- org.slf4j:slf4j-api:1.7.25
|    |    +--- org.springframework:spring-core:4.3.14.RELEASE (*)
|    |    \--- org.yaml:snakeyaml:1.17
|    +--- org.mongodb:mongodb-driver:3.4.3 -> 3.5.0 (*)
```

**Comment by ismaelgomescosta** [comment]
Yap. I think it's important to exclude this spring boot dependency to avoid conflicts

**Comment by bartoszwalacik** [comment]
version bump released in 3.8.5
